integrations/dynatrace/.port/spec.yaml

```diff
@@ -13,6 +13,8 @@ configurations:
     required: true
     type: string
     sensitive: true
+    description: "Dynatrace API Key with at least the following permissions: `DataExport`, `Read entities`, `Read problems` and `Read SLO`. Example: 'dt7c05.KA8AFCZXD299AJHEU134.MASEFA9ABDI9HAJI9HA83H8377AAJISURHHSHAUHFKSE83Z7' Head over to <instanceURL>/ui/apps/dynatrace.classic.tokens/ui/access-tokens to create one."
```

Comment:
Isn't there a dynatrace guide we can point them to so they will follow the official guide?